http_submission/sample_evaluation.py

```diff
@@ -0,0 +1,89 @@
+import numpy as np
+import pickle
+import requests
+import sys
+from sample_submission import MySimpleScanpathModel
+from PIL import Image
+from io import BytesIO
+import json
+import matplotlib.pyplot as plt
+from pysaliency.plotting import plot_scanpath
+sys.path.insert(0, '..')
+import pysaliency
+
+class HTTPScanpathModel(MySimpleScanpathModel):
+    def __init__(self, url):
+        self.url = url
+        self.log_density_url = url + "/conditional_log_density"
```

Comment:
I would suggest to use properties for that. Right now, if you change the url (for whatever reason), the dependent urls are not updated